spec/features/mechanics/show_spec.rb

```diff
@@ -0,0 +1,71 @@
+    @ride2 = @six_flags.rides.create!(name: 'The Scrambler', thrill_rating: 4, open: true)
+    @ride3 = @six_flags.rides.create!(name: 'Ferris Wheel', thrill_rating: 7, open: false)
+    
+    @ride4 = @universal.rides.create!(name: 'Jaws', thrill_rating: 5, open: true)
+    @ride5 = @universal.rides.create!(name: 'Jurrasic Park', thrill_rating: 8, open: true)
+    
+    @mechanic1 = Mechanic.create!(name: "Johny", years_experience: 10)
+    @mechanic2 = Mechanic.create!(name: "Billy", years_experience: 5)
+    
+    @mechanic_ride1 = MechanicRide.create!(mechanic: @mechanic1, ride: @ride1)
+    @mechanic_ride2 = MechanicRide.create!(mechanic: @mechanic1, ride: @ride2)
+    @mechanic_ride3 = MechanicRide.create!(mechanic: @mechanic1, ride: @ride3)
+    @mechanic_ride4 = MechanicRide.create!(mechanic: @mechanic2, ride: @ride4)
+    @mechanic_ride5 = MechanicRide.create!(mechanic: @mechanic2, ride: @ride5)
+    @mechanic_ride6 = MechanicRide.create!(mechanic: @mechanic1, ride: @ride4)
+  end
+
+  describe "As a visitor do" do
+    describe "when I visit a mechanics show page" do 
+      it "displays the name, years of experience, and names of all rides they are working on" do
+        visit mechanic_path(@mechanic1)
+
+        expect(page).to have_content("Mechanic #{@mechanic1.id}")
+        expect(page).to have_content("Name: #{@mechanic1.name}")
+        expect(page).to have_content("Years of Experience: #{@mechanic1.years_experience}")
+        expect(page).to have_content("Rides:")
+        expect(page).to have_content("#{@ride1.name}")
+        expect(page).to have_content("#{@ride2.name}")
+        expect(page).to have_content("#{@ride3.name}")
+        expect(page).to have_content("#{@ride4.name}")
+
+        expect(page).to_not have_content("Mechanic #{@mechanic2.id}")
+        expect(page).to_not have_content("Name: #{@mechanic2.name}")
+        expect(page).to_not have_content("Years of Experience: #{@mechanic2.years_experience}")
+      end
+
+      it "displays a form to add a ride to the mechanic" do
+        visit mechanic_path(@mechanic2)
+
+        expect(page).to have_content("Mechanic #{@mechanic2.id}")
+        expect(page).to have_content("Name: #{@mechanic2.name}")
+        expect(page).to have_content("Years of Experience: #{@mechanic2.years_experience}")
+        expect(page).to have_content("Rides:")
+        expect(page).to have_content("#{@ride4.name}")
+        expect(page).to have_content("#{@ride5.name}")
+
+        expect(page).to have_content("Add a ride to Mechanic:")
+
+        fill_in "ride_id", with: "#{@ride2.id}"
+
+        click_button "Submit"
+
+        expect(page).to have_current_path(mechanic_path(@mechanic2))
+
+        expect(page).to have_content("Rides:")
+        expect(page).to have_content("#{@ride2.name}")
+        expect(page).to have_content("#{@ride4.name}")
+        expect(page).to have_content("#{@ride5.name}")
```

Comment:
This works, and I like that you attempted to test that before the form, there were only 2 rides, but testing for the presence of other rides is NOT the same as testing that @ride2 is NOT on the page. On line 55, I'd love to see a line testing explicitly that @ride5.name is NOT on the page.